phabricatoremails/render/template.py

```diff
@@ -267,10 +274,11 @@ def _jinja_html(loader):
     jinja_env.filters["comment_summary"] = _comment_summary
     jinja_env.filters["secure_comment_summary"] = _secure_comment_summary
     jinja_env.globals["emoji"] = _emoji_html
+    jinja_env.globals["phabricator_host"] = phabricator_host
```

Comment:
Are settings accessible from this method? Wondering if it would be cleaner to fetch it directly from the settings, rather than passing this additional parameter to this method and to `TemplateStore.__init__`, since the impact ultimately is not functional but rather just affects a template variable.

Reply:
Hmm, I'm not sure I follow.
`settings` isn't available statically/globally since it needs to be initialized (the global variable would have to be `Optional`), and it doesn't look like it's passed into these functions.
Instead of passing `phabricator_host` down through these function calls, I could pass down an instance of `Settings`, but since just one property (`phabricator_host`) is used, I'm not sure that that is an improvement - besides, that could make tests slightly more work to write.

Unless you had something else in mind? Let me know what you're thinking :)

Reply:
I see -- given the way it is initialized, it doesn't make sense to re-initialize it again, so this is fine.

Something to keep in mind for the future, if we add more variables from `settings` -- I would recommend putting this and others in a "settings" dict (i.e. `jinja_env.globals["settings"] = {"phabricator_host": phabricator_host, ...}`) to avoid polluting the global namespace, and to add more context to these template variables from within the templates. We could even just have a tuple of whitelisted settings keys and include all of those in this dict -- which might help with debugging templates in the future.